### PR TITLE
Style collections view

### LINKED ISSUE #51: Changes to collections index view
- [x] Set a max-width on the content (this will be used everywhere but the mirador workspace view).
  * 100% width; max-width: 1280px; centered;
- [x] Make collection sections full width.  Turn actions links to buttons on the right.
  * Each collection should be visually distinct from one another (not just padding/margin)
- [ ] Add number of manifests.
- [x] Add number of workspaces instantiated.
- [x] Add date created.
- [ ] Add description (if we have it).

### PR DESCRIPTION
**Pull request by csbailey5t**
Fixes #51 

collection-index

### CODE REVIEW

File: app/assets/stylesheets/base.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.row {
```
Comment (mejackreed) [inline app/assets/stylesheets/base.scss]: What do you think about adding a new class here like `row-striped` ?

File: app/assets/stylesheets/collection.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.collection-result {
+  padding: 10px 5px;
```
Comment (jkeck) [inline app/assets/stylesheets/collection.scss]: I feel like there may be a variable that provides something similar to what we're trying to get at here, but not sure what that is.

File: app/assets/stylesheets/base.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  &.odd {
+    background-color: $table-bg-accent;
+  }
+}
+
+.content-area {
+  max-width: 1280px;
```
Comment (jkeck) [inline app/assets/stylesheets/base.scss]: Would this be worth putting into a shared variables file?

Comment (csbailey5t) [inline app/assets/stylesheets/base.scss]: That makes sense to me.

File: app/assets/stylesheets/base.scss (hunk 1 above)
Comment (csbailey5t) [inline app/assets/stylesheets/base.scss]: Sounds good. Gives us a bit more specificity by keeping us from adding too much onto the bootstrap classes.

**Pull request merged**